README.md

```diff
@@ -1,8 +1,9 @@
 # Azuki
 
-Azuki is generic framework designed to assist with Behaviour Driven Specification (BDS), in the same way that [Cucumber](https://cucumber.io/) assists with Behaviour driven development (BDD).
+Azuki is generic framework designed to assist with Behavior Driven Specification (BDS), in the same way that [Cucumber](https://cucumber.io/) assists with Behavior driven development (BDD).
```

Comment:
⛏️ You might as well fix the capitalisation of "Behavior Driven Development" while you're here.